documentation/guides/migrating-from-v7-to-v8.md

```diff
@@ -0,0 +1,3 @@
+# Migrating from v7 to v8
+
+Polaris v8.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/v8.0.0)) features a rebuild of the `Autocomplete` component, build changes, . This file describes all code updates required to stay up to date.
```

Comment:
Going to start creating a draft migration guide. Too lazy on Friday arvo to move it to another PR.